mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -61,7 +61,7 @@ CLASS axd net/minecraft/item/ItemStack
 		ARG 2 state
 		ARG 3 pos
 	METHOD a getBlockBreakingSpeed (Lbqe;)F
-	METHOD a areBlockProxiesEqual (Lbqi;Lbqi;)Z
+	METHOD a areBlockCachesEqual (Lbqi;Lbqi;)Z
```

Comment:
This should just be `areBlocksEqual`. The block position and world are not compared in this method, only the block and block entity are compared for equality.

Reply:
I agree. changing it right now.